src/shared/constants.ts

```diff
@@ -40,7 +42,7 @@ export const moviesList: IMovieInfo[] = [
     runtime: 135,
   },
   {
-    poster_path: "/movie-images/bohemian-rhapsody.png",
+    poster_path: commonImgUrl,
     title: "Bohemian Rhapsody",
     release_date: "2002-11-11",
```

Comment:
tests passed. But I still have another failed tests, JFYI

![image](https://github.com/user-attachments/assets/2d3df94b-f2df-449a-9b3b-7f993ab75969)

Reply:
Most likely, 1-2 UTs from this component fail randomly sometimes. I will fix them in the next module while working on the add/edit movie functionality
![image](https://github.com/user-attachments/assets/42720b26-d16b-4213-be40-75a8cc04a3f4)